cmd/ocm/config/reset/reset.go

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright (c) 2024 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package reset
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/openshift-online/ocm-sdk-go/authentication/securestore"
+)
+
+var args struct {
+	debug bool
+}
+
+var Cmd = &cobra.Command{
```

Comment:
If we are only supporting resetting keyring for now, we should mention that in the description

Reply:
Updated help text to allow additional context for each var